modules/migration-restic-errors.adoc

```diff
@@ -115,3 +115,32 @@ The output identifies the Restic Pod that logged the errors.
 ----
 $ oc logs -f restic-nr2v5
 ----
+
+ifeval::["{mtc-version}" < "1.3"]
+== Restic permission error for NFS storage
+
+If you are migrating data from NFS storage and `root_squash` is enabled, Restic maps to `nfsnobody` and does not have permission to perform the migration. The following error is displayed in the Restic Pod log.
+
+.Example output
+[source,terminal]
+----
+backup=openshift-migration/<backup_id> controller=pod-volume-backup error="fork/exec /usr/bin/restic: permission denied" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/controller/pod_volume_backup_controller.go:280" error.function="github.com/vmware-tanzu/velero/pkg/controller.(*podVolumeBackupController).processBackup" logSource="pkg/controller/pod_volume_backup_controller.go:280" name=<backup_id> namespace=openshift-migration
+----
+
+You can resolve this issue by creating a supplemental group for Restic and adding the group ID to the Migration Controller.
+
+.Procedure
+
+. Create a xref:../../storage/persistent_storage/persistent-storage-nfs.adoc#group-ids[supplemental group] using a common group.
```

Comment:
Unfortunately, we can't use `xref`s from within a module. You can move the link to an "Additional resources" list after this module in the assembly, however.

Reply:
Oops! Will fix

Reply:
Actually, I'll just take the xref out. If someone doesn't know how to create a supplemental group, they have no business doing a migration. :-)